test/controllers/project_skill_controller_test.exs

```diff
@@ -71,75 +60,49 @@ defmodule CodeCorps.ProjectSkillControllerTest do
 
       insert(:organization_membership, role: "admin", member: current_user, organization: organization)
 
-      payload = build_payload |> put_relationships(project, skill)
-      path = conn |> project_skill_path(:create)
-      json = conn |> post(path, payload) |> json_response(201)
-
-      id = json["data"]["id"] |> String.to_integer
-      project_skill = ProjectSkill |> Repo.get(id)
-
-      json
-      |> Map.get("data")
-      |> assert_result_id(project_skill.id)
-      |> assert_jsonapi_relationship("project", project.id)
-      |> assert_jsonapi_relationship("skill", skill.id)
-      |> assert_attributes(%{})
+      attrs = %{project: project, skill: skill}
+      assert conn |> request_create(attrs) |> json_response(201)
     end
 
     @tag authenticated: :admin
-    test "does not create resource and renders errors when data is invalid", %{conn: conn} do
-      payload = build_payload
-      path = conn |> project_skill_path(:create)
-      json = conn |> post(path, payload) |> json_response(422)
-
-      assert json["errors"] != %{}
+    test "renders 422 when data is invalid", %{conn: conn} do
+      assert conn |> request_create(@attrs) |> json_response(422)
```

Comment:
This looks like a case where we want `@invalid_attrs`, which is empty (no relationships).